3_visualize.R

```diff
@@ -58,46 +48,30 @@ p3_targets <- list(
                         dv_tibble = p3_dv_tibble,
                         growing_season = F)),
   
+  # Inset maps
+      # view = "CONUS" excludes Alaska and Hawaii
+      # view = "midwest" zooms in on midwest states
+  tar_target(p3_map_midwest,
+             inset_map(state_fill = "#1E466E",
+                       border_size = 0.05,
+                       border_color = "#5BA5B3",
+                       highlight_site_color = "#ec8176",
+                       view = "midwest")),
   tar_target(p3_inset_map,
-             inset_map(state_fill = "#1e466e",
-                       border_size = 0.1,
-                       border_fill = "#ddeff2",
-                       highlight_site_color = "#fcee21")),
+             inset_map(state_fill = "#1E466E",
+                       border_size = 0.05,
+                       border_color = "#1E466E",
+                       highlight_site_color = "#fcee21",
+                       view = "CONUS")),
 
-    # Bottom bars are specified in four ways:
-      # extent = "fixed one" when only the August fixed threshold drought should show; growing season time frame
-      # extent = "fixed all" when all the fixed threshold droughts can show; growing season time frame
-      # extent = "both" when all the droughts can show; growing season time frame
-      # extent = "both year" when all the droughts can show; full year time frame
-  tar_target(p3_bottom_bars_fixed_one,
-             bottom_bars(extent = "fixed one", 
-                         droughts_df = p2_droughts_learner_viz_df, 
-                         blank_plot = p3_blank_plot_summer,
-                         dv_tibble = p3_dv_tibble)),
-  tar_target(p3_bottom_bars_fixed_all,
-             bottom_bars(extent = "fixed all", 
-                         droughts_df = p2_droughts_learner_viz_df, 
-                         blank_plot = p3_blank_plot_summer,
-                         dv_tibble = p3_dv_tibble)),
-  tar_target(p3_bottom_bars_both,
-             bottom_bars(extent = "both", 
-                         droughts_df = p2_droughts_learner_viz_df, 
-                         blank_plot = p3_blank_plot_summer,
-                         dv_tibble = p3_dv_tibble)),
-  tar_target(p3_bottom_bars_both_year,
-             bottom_bars(extent = "both year", 
-                         droughts_df = p2_droughts_learner_viz_df, 
-                         blank_plot = p3_blank_plot_summer,
-                         dv_tibble = p3_dv_tibble)),
   
   
   # Frames 
   tar_target(p3_drought_learner_viz_a_png,
              frame_a(blank_plot = p3_blank_plot_summer,
```

Comment:
Not to address in this PR, but in the next phase let's reduce the number of unique functions for similar plots.